src/zui/components/ZUIVerticalBarChart/ZUIVerticalBarChartScale.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Typography } from '@mui/material';
+import React, { FC } from 'react';
+import { useTheme } from '@mui/system';
+
+export interface ZUIVerticalBarChartScaleProps {
```

Comment:
Same as in the other, no need to export this, and we are currently using `type` rather than `interface` when it is a type. 

Also naming, `Scale` I think would be clear enough for this component